cvat/apps/analytics_report/report/get.py

```diff
@@ -0,0 +1,127 @@
+    statistics = _convert_datetime_to_date(statistics)
+
+    if obj_model is Job:
+        target = TypeChoice.JOB
+    elif obj_model is Task:
+        target = TypeChoice.TASK
+    elif obj_model is Project:
+        target = TypeChoice.PROJECT
+
+    data = {
+        "target": target,
+        f"{obj_model.__name__.lower()}_id": pk,
+        "statistics": statistics,
+        "created_date": db_analytics_report.created_date,
+    }
+    return data
+
+def _get_job_report(job_id, start_date, end_date):
+    return _get_object_report(Job, int(job_id), start_date, end_date)
+
+def _get_task_report(task_id, start_date, end_date):
+    return _get_object_report(Task, int(task_id), start_date, end_date)
+
+def _get_project_report(project_id, start_date, end_date):
+    return _get_object_report(Project, int(project_id), start_date, end_date)
+
+def get_analytics_report(request, query_params):
+    query_params = {
+        'project_id': query_params.get('project_id', None),
+        'task_id': query_params.get('task_id', None),
+        'job_id': query_params.get('job_id', None),
+        'start_date': query_params.get('start_date', None),
+        'end_date': query_params.get('end_date', None),
+    }
+
+    try:
+        if query_params['start_date']:
+            query_params['start_date'] = parser.parse(query_params['start_date'])
+    except parser.ParserError:
+        raise serializers.ValidationError(
+            f"Cannot parse 'start_date' datetime parameter: {query_params['start_date']}"
+        )
+    try:
+        if query_params['end_date']:
+            query_params['end_date'] = parser.parse(query_params['end_date'])
+    except parser.ParserError:
+        raise serializers.ValidationError(
+            f"Cannot parse 'end_date' datetime parameter: {query_params['end_date']}"
+        )
+
+    if query_params['start_date'] and query_params['end_date'] and query_params['start_date'] > query_params['end_date']:
+        raise serializers.ValidationError("'start_date' must be before than 'end_date'")
+
+    # Set the default time interval to last 30 days
+    if not query_params["start_date"] and not query_params["end_date"]:
+        query_params["end_date"] = datetime.now(timezone.utc)
+        query_params["start_date"] = query_params["end_date"] - timedelta(days=30)
+    elif query_params["start_date"] and not query_params["end_date"]:
+        query_params["end_date"] = datetime.now(timezone.utc)
+    elif not query_params["start_date"] and query_params["end_date"]:
+        query_params["end_date"] = datetime.min
+
+    job_id = query_params.get('job_id', None)
+    task_id = query_params.get('task_id', None)
+    project_id = query_params.get('project_id', None)
+
+    if job_id is None and task_id is None and project_id is None:
+        raise serializers.ValidationError("No any job, task or project specified")
+
+    if [job_id, task_id, project_id].count(True) > 1:
```

Comment:
Probably, this line works differently than expected:
```
[1, 2, 0].count(True)
>>> 1
```

```suggestion
    if sum(map(bool, [job_id, task_id, project_id])).count(True) > 1:
```

```suggestion
    if not (bool(job_id) ^ bool(task_id) ^ bool(project_id)):
```